draft-ietf-taps-transport-security.md

```diff
@@ -844,6 +855,70 @@ and the per-message nonce in the clear. Everything else is encrypted.
 
 - An unreliable transport protocol such as UDP.
 
+## OpenVPN
+
+OpenVPN {{OpenVPN}} is a commonly used protocol designed to replace IPsec. A
+major goal of this protocol is to provide an easy-to-use VPN. OpenVPN
+encapsulates either IP packets or Ethernet frames within a secure tunnel and
+can run over UDP or TCP.
+
+### Protocol Description
+
+OpenVPN facilitates authentication using either a pre-shared static key or
+using X.509 certificates and TLS. In pre-shared key mode, OpenVPN derives
+keys for encryption and authentication directly from one or multiple static
+keys. In TLS mode, OpenVPN encapsulates a TLS handshake, in which both peers
+must present a certificate for authentication. After the handshake, both sides
+contribute random source material to derive keys for encryption and
+authentication using the TLS pseudo random function (PRF). OpenVPN provides the
+possibility to authenticate and encrypt the TLS handshake itself using a
+pre-shared key or passphrase. Furthermore, it supports rekeying using TLS.
+
+After authentication and key exchange, OpenVPN encrypts payload data, i.e., IP
+packets or Ethernet frames, and signs the payload using an HMAC function. The
+default cipher is BlowFish and the default message digest algorithm is SHA1,
+but an application can select an arbitrary cipher, key size, and message digest
+algorithm for the HMAC. OpenVPN peers may also support cipher negotiation
+(NCP). If both hosts allow, they can upgrade the cipher to AES-256-GCM.
+Blocks are padded to an even multiple of block size.
+
+OpenVPN can run over TCP or UDP. When running over UDP, OpenVPN provides a
+simple reliability layer for control packets such as the TLS handshake and key
+exchange. It assigns sequence numbers to packets, acknowledges packets it
+receives, and retransmits packets it deems lost. This reliability layer is not
```

Comment:
nit: Possible add, "Similarly to DTLS, this reliability layer..."

Reply:
Agreed!

Reply:
👍